refactor(forms): remove self closing

PatrickJS opened the pull request:
html5 self close tags while xhtml you have to have self closing tags
breaks `stringifyElement` from https://github.com/angular/angular/pull/2106

also fixed typos
https://github.com/angular/angular/blob/master/modules/examples/src/forms/index.ts#L190
https://github.com/angular/angular/blob/master/modules/examples/src/forms/index.ts#L191
https://github.com/angular/angular/blob/master/modules/examples/src/forms/index.ts#L192

and wrapped `<input>` in `<label>`

vicb commented on modules/examples/src/forms/index.ts:
```diff
@@ -21,15 +21,15 @@ import {print} from 'angular2/src/facade/lang';
         <div>
-          <label>Title:</label> <br/>
-          <input type="text" control="title"/>
+          <label>Title:</label> <br>
```
While fixing this, what about
```html
<label>
  Title:
  <input type="text" control="title">
</label>
```

gdi2290 replied:
yeah, give me a few minutes to fix and rebase

gdi2290 commented:
@vicb alright I correctly wrapped the element in <label> and rebase'd

PatrickJS committed:
refactor(forms): remove self closing wrap <label>s

html5 doesn't have self closing tags
breaks `stringifyElement` from
https://github.com/angular/angular/pull/2106

also an error with submit form and wrap with <label>s

vicb commented:
merged via c19c69f336ddb3b64584f0ecb8d09b7417a3bce1
